frontend/src/metabase/account/settings/selectors.js

```diff
@@ -0,0 +1 @@
+export const getPath = (state, props) => props.location.pathname;
```

Comment:
Looks like this function is used in a single place. Should we maybe just inline it there? Also, when I see a "selector" in a React-Redux app, I personally expect it to return something from the store, so this could be a little bit confusing. Maybe it's just a preconception. What do you think about that?

Reply:
Agree, removed